gap_darwin.go

```diff
@@ -162,5 +162,9 @@ func (pd *peripheralDelegate) DidUpdateValueForCharacteristic(prph cbgo.Peripher
 		if char != nil && char.callback != nil {
 			go char.callback(chr.Value())
 		}
```

Comment:
This means that if you're also listening for notifications, you'll get a signal for a notification while no notification happened.

This is unexpected, but I don't see an easy way to work around that in CoreBluetooth: `DidUpdateValueForCharacteristic` does not distinguish between notification updates and the result of reading a value.

Reply:
It is probably not that likely for developer to use both a notify and also a separate read for the same characteristic is it? That seems like mixing metaphors at least a little.

As you point out, that is not something that can be dealt with on macOS it seems.